ecowitt2mqtt/hass.py

```diff
@@ -134,6 +140,20 @@ class EntityDescription:
         device_class=DEVICE_CLASS_TEMPERATURE,
         unit_class=UNIT_CLASS_TEMPERATURE,
     ),
+    DATA_POINT_LIGHTNING: EntityDescription(
+        platform=PLATFORM_SENSOR,
+        icon="mdi:weather-lightning",
+        unit=UNIT_STRIKES,
+    ),
+    DATA_POINT_LIGHTNING_NUM: EntityDescription(
+        platform=PLATFORM_SENSOR,
+        icon="mdi:map-marker-distance",
+        unit=UNIT_KM,
```

Comment:
Can we convert units based on the setting of `OUTPUT_UNIT_SYSTEM`? So if imperial convert to miles and if metric we can keep as km.

I spoke to ecowitt support and the unit is always km regardless of the unit settings on the device.